### ISSUE TITLE
Have search service work in staging and production

### ISSUE THREAD

**Issue by HyperTHD** [Seneca-CDOT/telescope#2014]
Once #2106 lands, we'll need to setup the staging and production docker files to use this new service as well as ensure it's used in the frontend.

**Comment by HyperTHD** [comment]
#2012 was bugged, and rebasing master onto it ended up closing the PR. 
Gonna have another PR to fix this now that #2106 is landed
